return {
    ["Andromeda"] = require(script.Parent["Andromeda"].table),
    ["Antlia"] = require(script.Parent["Antlia"].table),
    ["Apus"] = require(script.Parent["Apus"].table),
    ["Aquarius"] = require(script.Parent["Aquarius"].table),
    ["Aquila"] = require(script.Parent["Aquila"].table),
    ["Ara"] = require(script.Parent["Ara"].table),
    ["Aries"] = require(script.Parent["Aries"].table),
    ["Auriga"] = require(script.Parent["Auriga"].table),
    ["Boötes"] = require(script.Parent["Boötes"].table),
    ["Caelum"] = require(script.Parent["Caelum"].table),
    ["Camelopardalis"] = require(script.Parent["Camelopardalis"].table),
    ["Cancer"] = require(script.Parent["Cancer"].table),
    ["Canes Venatici"] = require(script.Parent["Canes Venatici"].table),
    ["Canis Major"] = require(script.Parent["Canis Major"].table),
    ["Canis Minor"] = require(script.Parent["Canis Minor"].table),
    ["Capricornus"] = require(script.Parent["Capricornus"].table),
    ["Carina"] = require(script.Parent["Carina"].table),
    ["Cassiopeia"] = require(script.Parent["Cassiopeia"].table),
    ["Centaurus"] = require(script.Parent["Centaurus"].table),
    ["Cepheus"] = require(script.Parent["Cepheus"].table),
    ["Cetus"] = require(script.Parent["Cetus"].table),
    ["Chamaeleon"] = require(script.Parent["Chamaeleon"].table),
    ["Circinus"] = require(script.Parent["Circinus"].table),
    ["Columba"] = require(script.Parent["Columba"].table),
    ["Coma Berenices"] = require(script.Parent["Coma Berenices"].table),
    ["Corona Australis"] = require(script.Parent["Corona Australis"].table),
    ["Corona Borealis"] = require(script.Parent["Corona Borealis"].table),
    ["Corvus"] = require(script.Parent["Corvus"].table),
    ["Crater"] = require(script.Parent["Crater"].table),
    ["Crux"] = require(script.Parent["Crux"].table),
    ["Cygnus"] = require(script.Parent["Cygnus"].table),
    ["Delphinus"] = require(script.Parent["Delphinus"].table),
    ["Dorado"] = require(script.Parent["Dorado"].table),
    ["Draco"] = require(script.Parent["Draco"].table),
    ["Equuleus"] = require(script.Parent["Equuleus"].table),
    ["Eridanus"] = require(script.Parent["Eridanus"].table),
    ["Fornax"] = require(script.Parent["Fornax"].table),
    ["Gemini"] = require(script.Parent["Gemini"].table),
    ["Grus"] = require(script.Parent["Grus"].table),
    ["Hercules"] = require(script.Parent["Hercules"].table),
    ["Horologium"] = require(script.Parent["Horologium"].table),
    ["Hydra"] = require(script.Parent["Hydra"].table),
    ["Hydrus"] = require(script.Parent["Hydrus"].table),
    ["Indus"] = require(script.Parent["Indus"].table),
    ["Lacerta"] = require(script.Parent["Lacerta"].table),
    ["Leo"] = require(script.Parent["Leo"].table),
    ["Leo Minor"] = require(script.Parent["Leo Minor"].table),
    ["Lepus"] = require(script.Parent["Lepus"].table),
    ["Libra"] = require(script.Parent["Libra"].table),
    ["Lupus"] = require(script.Parent["Lupus"].table),
    ["Lynx"] = require(script.Parent["Lynx"].table),
    ["Lyra"] = require(script.Parent["Lyra"].table),
    ["Mensa"] = require(script.Parent["Mensa"].table),
    ["Microscopium"] = require(script.Parent["Microscopium"].table),
    ["Monoceros"] = require(script.Parent["Monoceros"].table),
    ["Musca"] = require(script.Parent["Musca"].table),
    ["Norma"] = require(script.Parent["Norma"].table),
    ["Octans"] = require(script.Parent["Octans"].table),
    ["Ophiuchus"] = require(script.Parent["Ophiuchus"].table),
    ["Orion"] = require(script.Parent["Orion"].table),
    ["Pavo"] = require(script.Parent["Pavo"].table),
    ["Pegasus"] = require(script.Parent["Pegasus"].table),
    ["Perseus"] = require(script.Parent["Perseus"].table),
    ["Phoenix"] = require(script.Parent["Phoenix"].table),
    ["Pictor"] = require(script.Parent["Pictor"].table),
    ["Pisces"] = require(script.Parent["Pisces"].table),
    ["Piscis Austrinus"] = require(script.Parent["Piscis Austrinus"].table),
    ["Puppis"] = require(script.Parent["Puppis"].table),
    ["Pyxis"] = require(script.Parent["Pyxis"].table),
    ["Reticulum"] = require(script.Parent["Reticulum"].table),
    ["Sagitta"] = require(script.Parent["Sagitta"].table),
    ["Sagittarius"] = require(script.Parent["Sagittarius"].table),
    ["Scorpius"] = require(script.Parent["Scorpius"].table),
    ["Sculptor"] = require(script.Parent["Sculptor"].table),
    ["Scutum"] = require(script.Parent["Scutum"].table),
    ["Serpens"] = require(script.Parent["Serpens"].table),
    ["Sextans"] = require(script.Parent["Sextans"].table),
    ["Taurus"] = require(script.Parent["Taurus"].table),
    ["Telescopium"] = require(script.Parent["Telescopium"].table),
    ["Triangulum"] = require(script.Parent["Triangulum"].table),
    ["Triangulum Australe"] = require(script.Parent["Triangulum Australe"].table),
    ["Tucana"] = require(script.Parent["Tucana"].table),
    ["Ursa Major"] = require(script.Parent["Ursa Major"].table),
    ["Ursa Minor"] = require(script.Parent["Ursa Minor"].table),
    ["Vela"] = require(script.Parent["Vela"].table),
    ["Virgo"] = require(script.Parent["Virgo"].table),
    ["Volans"] = require(script.Parent["Volans"].table),
    ["Vulpecula"] = require(script.Parent["Vulpecula"].table),
}


